src/components/UploadSong.tsx

```diff
@@ -63,7 +63,7 @@ const UploadSong: FunctionComponent<UploadSongProps> = ({
     onDrop: handleDrop,
     multiple: false,
     accept: {
-      audio: [".flac", ".fla", ".wav"],
+      audio: [".flac", ".wav"],
```

Comment:
Removing fla to avoid confusion as suggested here: https://projectnewmgroup.slack.com/archives/C04P2C60JS1/p1680618743534499